azure-batch/HISTORY.rst

```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+5.0.0 (2018-08-24)
+++++++++++++++++++
+
+- Using REST API version 2018-08-01.7.0.
+- [Breaking] Model signatures are now using only keywords-arguments syntax. Every positional arguments are required to be rewritten as keywords-arguments.
```

Comment:
You missed a few other breaking changes in the REST API itself. 

- **[Breaking]** Removed the `ValidationStatus` property from [TaskCounts](rest/api/batchservice/job/gettaskcounts).
- **[Breaking]** The default caching type for [DataDisk](https://docs.microsoft.com/en-us/rest/api/batchservice/pool/add#datadisk) and [OSDisk](https://docs.microsoft.com/en-us/rest/api/batchservice/pool/add#osdisk) is now `ReadWrite` instead of `None`.

(Fix up to make sense in Python -- above are from REST API itself

Reply:
Done.